src/Admin/Views/Providers/View.cshtml

```diff
@@ -1,11 +1,32 @@
-﻿@model ProviderViewModel
+﻿@inject Bit.Core.Settings.GlobalSettings GlobalSettings
+@model ProviderViewModel
 @{
     ViewData["Title"] = "Provider: " + Model.Provider.Name;
```

Comment:
I might be missing something, but are we using this anywhere?

Reply:
I believe it's used in _Layout.cshtml to populate the bowser tab title:
`<title>@ViewData["Title"] - Bitwarden Admin Portal</title>`